### DISCUSSION TITLE
Export only surface particles

### DISCUSSION THREAD

**Discussion by Perpetual3134** [InteractiveComputerGraphics/SPlisHSPlasH#316] [Q&A]
Is there a way to export only particles, which are representing the surface?

**Comment by janbender** [comment]
No, not yet. But you can implement your own exporter easily.

**Comment by Perpetual3134** [comment]
do i understand correctly that i can use the macro forall_boundary_neighbors to loop over all particles that are on the surface?

**Comment by janbender** [comment]
Yes. However, this is only possible if you use the Akinci boundary handling. The other methods do not use boundary particles.
